root/static/scripts/common/components/PregapTrackIcon.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * @flow strict
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+const PregapTrackIcon = (): React.Element<'div'> => (
+  <div
+    className="pregap-track icon img"
+    title={l('This track is hidden in the pregap.')}
+  />
+);
+
+export default PregapTrackIcon;
```

Comment:
That should probably be a separate commit. I found this macro conversion to be made in pull request https://github.com/metabrainz/musicbrainz-server/pull/1660 (commit https://github.com/metabrainz/musicbrainz-server/pull/1660/commits/17ef466f4caabbbf4db4f19c6392f16c36a41131#diff-8ea75b5338e161bca6ed99b878adb716R847) too.

Reply:
Split the commit :)

Reply:
Thanks!